apps/admin-gui/src/app/shared/components/member-overview-groups/member-overview-groups.component.html

```diff
@@ -30,14 +30,14 @@ <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.GROUP_MEMBERSHIP' | translat
           <td *matCellDef="let attribute" class="column-center" mat-cell>
             <div *ngIf="attribute === 'Status'">
               <i
-                class="material-icons vert-center mr-1 {{(selectedMember.groupStatus | memberStatusIconColor)}}">
+                class="material-icons vert-center mr-1 {{(selectedMember | memberStatusIconGroupColor)}}">
                 {{selectedMember.groupStatus | memberStatusIcon}}
               </i>
               <b class="vert-top">
                 {{selectedMember.groupStatus | transformMemberStatus}}
               </b>
               <button
-                *ngIf="authResolver.isThisVoAdmin(vo.id) || authResolver.isThisGroupAdmin(this.selectedGroup.id)"
+                *ngIf="(authResolver.isThisVoAdmin(vo.id) || authResolver.isThisGroupAdmin(this.selectedGroup.id)) && !(member | memberListCheckboxDisabled)"
```

Comment:
I did not realize before, but in this task the focus was to prevent changes on `Vo` level to those members of **parent organization** that are "mirrored" from **member organizations**. This does not apply to changes within `Group`. So here changing status/expiration  should be allowed.